publication-model/src/main/java/no/unit/nva/model/instancetypes/PublicationInstance.java

```diff
@@ -68,6 +69,7 @@
 
 @JsonTypeInfo(use = JsonTypeInfo.Id.NAME, property = "type")
 @JsonSubTypes({
+    @JsonSubTypes.Type(name = "AcademicCommentary", value = AcademicCommentary.class),
```

Comment:
AcademicCommentary is a fresh InstanceType that has been added and mapped in last cristin-import run.